### PR TITLE
[ To Main ] - DESENG-543 - Adding fontawesome pro to met-web

### PR DESCRIPTION
**Pull request by ratheesh-aot**
Issue #: https://apps.itsm.gov.bc.ca/jira/browse/DESENG-543

*Description of changes:*

  - Added font awesome to MET web.
  - Created .npmrc file to set the registry for forntawesome
  - Updated github actions to pull fontawesome auth token from github secrets

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the met-public license (Apache 2.0).

### CODE REVIEW

File: met-web/public/index.html [hunk 1]
```diff
@@ -8,8 +8,6 @@
         <meta name="description" content="Web site of MET" />
         <script src="%PUBLIC_URL%/config/config.js"></script>
         <script src="/snowplow.js" charset="UTF-8"></script>
-        <!--TODO: Remove font awesome cdn script when we have icons svgs stored in DB-->
-        <script src="https://kit.fontawesome.com/a2e8d2c4a8.js" crossorigin="anonymous"></script>
```
Comment (NatSquared) [inline met-web/public/index.html:12]: nice :)

Comment (NatSquared) [inline met-web/public/index.html:12]: nice :)

File: met-web/src/components/engagement/form/EngagementFormTabs/EngagementContent/ContentTabs.tsx [hunk 2]
```diff
@@ -1,7 +1,8 @@
 import React, { useContext, useState } from 'react';
 import { Box, Button, IconButton, MenuItem, Menu, Skeleton, Grid, Divider } from '@mui/material';
 import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
-import { faTrash, faPenToSquare, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faTrash, faPlus } from '@fortawesome/free-solid-svg-icons';
+import { faSquarePen } from '@fortawesome/pro-regular-svg-icons';
```
Comment (NatSquared) [inline met-web/src/components/engagement/form/EngagementFormTabs/EngagementContent/ContentTabs.tsx:5]: Could we import faTrash and faPlus from the pro tier as well? or do they only exist in the free package

**Review by NatSquared — APPROVED** [review]
Looks good to me!

File: met-web/src/components/engagement/form/EngagementFormTabs/EngagementContent/ContentTabs.tsx (hunk 2 above)
Comment (ratheesh-aot) [reply met-web/src/components/engagement/form/EngagementFormTabs/EngagementContent/ContentTabs.tsx:5]: this change I did to just to confirm whether the pro-integration is working. We may have another ticket to change whole icons to pro or in the future we can both

**Pull request merged**